### PR TITLE
Added audio as an optional control #12

### PR DESCRIPTION
**Pull request by i-slither**
Added audio as an optional control.

### CODE REVIEW

File: views/index.html [hunk 1]
```diff
@@ -68,4 +75,4 @@
       </div>
     </div>
   </body>
-</html>
+</html>
```
Comment (Srushtika) [inline views/index.html:78]: Please ensure all files have a newline at the end, see https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file for more info on the reasoning.

File: public/style.css [hunk 2]
```diff
@@ -291,4 +307,4 @@ label {
 
 .create-wait-msg {
   display: none;
-}
+}
```
Comment (Srushtika) [inline public/style.css:310]: Same comment re the newline as in the index.html file

File: views/index.html [hunk 3]
```diff
@@ -53,6 +53,13 @@
         <div id="room-code" class="col">Game room 0</div>
         <div id="join-leave-updates" class="col"></div>
         <div id="deaths" class="col"></div>
+        <div class="sound-control">
+          <label class="sound-label" for="sound">Sound</label>
+          <input class="sound-checkbox" type="checkbox" id="sound" name="toggle-sound" value="" />
```
Comment (Srushtika) [inline views/index.html:58]: Please replace all occurences of sound -> audio as a more standard way of refrence it. Please update the id to be more meaningful, for example, `audio-chkbox`. This is being referenced in the javascript file, so it needs to make it clear what is being referenced. Also, remove the attributes you don't need, like the value attribute which is empty.

File: public/script.js [hunk 4]
```diff
@@ -18,6 +18,7 @@ const myNickname = localStorage.getItem("nickname");
 const myGameRoomCode = localStorage.getItem("roomCode");
 const amIHost = localStorage.getItem("isHost");
 const startGameBtn = document.getElementById("btn-startgame");
+const soundControl = document.getElementById("sound");
```
Comment (Srushtika) [inline public/script.js:21]: This variable name doesn't clarify what it holds. It should be `audioControlChkBox`

**Review by Srushtika — CHANGES_REQUESTED** [review]
Thanks for your PR, it looks great and works as expected! Nice work. 

I've requested a few small changes in terms of improving the coding standards. As a general note, please replace all occurences of `sound` with `audio` as it's a more standard way.

File: views/index.html (hunk 1 above)
Comment (i-slither) [reply views/index.html:78]: O.k

**Review by Srushtika — APPROVED** [review]
Looks great, thank you! 🎉

**Pull request merged**